src/packagenamepy/__init__.py

```diff
@@ -1,5 +1,12 @@
-"""Short description of the package."""
-try:
-    from ._version import __version__  # noqa: F401
-except ImportError:
-    __version__ = "unknown"
+"""
+Contains the entry point for the application
+"""
+
+from ._version import __version__  # noqa: F401
+
```

Comment:
what is the reason for dropping try-except block here?  `._version` is a generated file, and it should not be part of git.

Reply:
Don't know why that got removed